core/server/services/slack.js

```diff
@@ -48,11 +54,51 @@ function ping(post) {
             return;
         }
 
+        var blogUrl = config.get('url');
+        var blogTitle = settingsCache.get('title');
+
         slackData = {
-            text: message,
+            // We are handling the case of test notification here by checking
+            // if it is a post or a test message to check webhook working.
+            text: schema.isPost(post) ? `Notification from *${blogTitle}* :ghost:` : message,
```

Comment:
Thinking that putting this `text` variable and the `attachments` below into `if (shema.isPost(...` above or separate if just above `slackData` would make things more readable and would follow the convention in this module :wink: